sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/display/DisplayDataTest.java

```diff
@@ -379,6 +379,7 @@ public void testAddIfNotNull() {
       public void populateDisplayData(Builder builder) {
         builder
             .addIfNotNull(DisplayData.item("nullString", (String) null))
+            .addIfNotNull(DisplayData.item("nullVPString", (ValueProvider<String>) null))
```

Comment:
Should it also handle `addIfNotNull(DisplayData.item("nullVPValue", ValueProvider.of((String) null))` ?